### PR TITLE
Fist draft of PastML meta.yaml

### PR DESCRIPTION
**Pull request by annazhukova**
Add pastml recipe 

-------------------------

Info | Link or Description
-- | --
Recipe | recipes/pastml
Summary | Ancestral character reconstruction for rooted phylogenetic trees.
Home | https://pastml.pasteur.fr
Releases | https://github.com/evolbioinfo/pastml/releases
Recipe Maintainer(s) | @annazhukova

### CODE REVIEW

**Comment by annazhukova** [comment]
@BiocondaBot please add pastml

**Review by andersgs — CHANGES_REQUESTED** [review]
Interesting tool. I'll have to try it out. I think it is probably a good idea to set minimum version for Python, numpy, scipy, and pandas. It will help avoid lengthy conda installs. And biopython. Some newer version have features that break in older versions.

**Comment by annazhukova** [comment]
Thanks @andersgs! I have added minimum versions.

File: recipes/pastml/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  number: 0
+  noarch: python
+  entry_points:
+    - pastml = pastml.acr:main
+    - geomap = pastml.visualisation.generate_geomap:main
+    - transition_counter = pastml.utilities.transition_counter:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
```
Comment (andersgs) [inline recipes/pastml/meta.yaml:22]: I was just reviewing the host requirements. I believe you only need to list python and pip here. And in run below, you don't need pip.

**Review by andersgs — CHANGES_REQUESTED** [review]
Sorry @annazhukova. I added one more request. I was just reviewing the host/run requirements here: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#host

File: recipes/pastml/meta.yaml (hunk 1 above)
Comment (annazhukova) [reply recipes/pastml/meta.yaml:22]: Indeed, the package dependencies  were only needed for run and not host. Thank you for spotting it @andersgs ! I have now fixed them

**Review by andersgs — APPROVED** [review]

**Comment by andersgs** [comment]
@BiocondaBot please merge

**Pull request merged**